docs/src/main/sphinx/connector/hudi.rst

```diff
@@ -0,0 +1,152 @@
+==============
+Hudi connector
+==============
+
+.. raw:: html
+
+  <img src="../_static/img/hudi.png" class="connector-logo">
+
+The Hudi connector enables querying `Hudi <https://hudi.apache.org/docs/overview/>`_
+tables synced to Hive metastore. The connector uses the metastore only to track
+partition locations. It makes use of the underlying Hudi filesystem and input formats
+to list data files. To learn more about the design of the connector, please check out
```

Comment:
This documentation is primarily intended to Trino users and not Trino contributors.

Please consider adding the design details of the connector in a `README.md` under `trino-hudi` module.